com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -277,6 +303,17 @@ static void CheckPersistencyPopupAlreadyOpened()
         [Callbacks.DidReloadScripts]
         static void WizardBehaviour()
         {
+            // If the wizard does not need to be shown at start up, do nothing.
+            if (!HDProjectSettings.wizardIsStartPopup)
+                return;
+
+            // Specify that the wizard has not been shown
+            // Set it here as the value stored on the settings might be wrong if Unity has stopped in scenarios like:
+            // - The user might stop the engine process.
+            // - The developer has ended the process from the engine side while attached
+            // - The editor crashed and that setting was not saved
+            HDUserSettings.wizardPopupAlreadyShownOnce = false;
```

Comment:
The main intent of this boolean was to not reload script on domain reload.

Please check that domain reload will not trigger the pop up (when wizard is setup to pop at start)

Reply:
If this is the case, we prefer having this small issue that only occures as a result of a crash.
Having the popup popping at each domain reload is a way higher issue.

Reply:
Ok, I didnt know that, removed that line of code :)